plugin/Renderer/InputFieldCheckboxRenderer.php

```diff
@@ -136,4 +144,12 @@ public function setCheckedValues($checkedValues)
 	/** @return array */
 	public function getCheckedValues()
 		{ return $this->_checkedValues; }
+
+	/**
+	 * @param Container $pContainer
+	 */
+	public function setPContainer(Container $pContainer)
+	{
+		$this->_pContainer = $pContainer;
```

Comment:
The setter should be removed

Reply:
I see this is hard. Could this be accomplished by making the container an optional constructor argument?